{"name": "TC3_002: Log In with invalid credentials via UI", "status": "passed", "steps": [{"name": "Step 1: Enter login data", "status": "passed", "attachments": [{"name": "image", "source": "a4c95eab-8d76-402f-b0c4-9c200574fcc4-attachment.png", "type": "image/png"}], "start": 1737989150131, "stop": 1737989150836}, {"name": "Step 2: Click Login Button", "status": "passed", "attachments": [{"name": "image", "source": "18d0bb2a-f061-4053-983d-d6d273063d6f-attachment.png", "type": "image/png"}], "start": 1737989150836, "stop": 1737989151215}, {"name": "Step 3: Verifying Failed Login", "status": "passed", "attachments": [{"name": "image", "source": "244fc015-1680-4e23-86a6-ac5737fa37d7-attachment.png", "type": "image/png"}], "start": 1737989151216, "stop": 1737989152364}], "attachments": [{"name": "log", "source": "f0f87f8a-5499-47a2-a2f8-14f659349bbf-attachment.txt", "type": "text/plain"}], "start": 1737989149798, "stop": 1737989152365, "uuid": "85245f51-1c0d-4181-bd9b-addcdad1de18", "historyId": "6cbd45f9c6807a29e9f1c493e0094cfe", "testCaseId": "6cbd45f9c6807a29e9f1c493e0094cfe", "fullName": "tests.ui.test_login_via_ui.TestUserLoginFunctionality#test_login_with_invalid_credential_via_ui", "labels": [{"name": "subSuite", "value": "TC3_002: Log In with invalid credentials via UI"}, {"name": "severity", "value": "normal"}, {"name": "suite", "value": "User Login Functionality"}, {"name": "feature", "value": "User Login Functionality"}, {"name": "parentSuite", "value": "Regression"}, {"name": "tag", "value": "tags('ui_login_functionality')"}, {"name": "host", "value": "CHEQ-J7M1HB3"}, {"name": "thread", "value": "13628-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.ui.test_login_via_ui"}]}